benches/wasm-kernel/src/regex_redux.rs

```diff
@@ -0,0 +1,17 @@
+//! Initially it supposed to be like [1]. However it turned out
+//! that executing this code in wasmi way too slow.
+//!
+//! [1]: https://benchmarksgame-team.pages.debian.net/benchmarksgame/program/regexredux-rust-2.html
+
+lazy_static! {
+	static ref REGEX: ::regex::bytes::Regex =
+		{ ::regex::bytes::Regex::new("agggtaa[cgt]|[acg]ttaccct").unwrap() };
+}
+
+pub fn prepare() {
+	&REGEX;
```

Comment:
This should be `&*REGEX`, AFAIK currently this doesn't do anything. Even better:

```rust
pub fn prepare() {
    // Force initialization of REGEX
    ::lazy_static::initialize(&REGEX);
}
```

Using the [`initialize` function in `lazy_static`](https://docs.rs/lazy_static/1.0.1/lazy_static/fn.initialize.html)

Reply:
Oh, that's really nice!

Reply:
This one is really nice. It is much cleaner than obscure `&REGEX`/`&*REGEX`